src/components/Admin/Judges.jsx

```diff
@@ -137,6 +137,7 @@ const Judges = () => {
         objects={filteredJudges}
         filters={filters}
         reset={judges}
+        file="JUDGES.csv"
```

Comment:
apologies, but now that we r doing the more complex naming scheme, u can keep the .csv at the final stage with the env variables and stuff. just need JUDGES here.